tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "outDir": "build",
     "downlevelIteration": true,
     "declaration": true,
-    "lib": ["es6", "dom", "dom.iterable", "es2018.promise"]
+    "lib": ["es6", "dom", "dom.iterable", "es2018.promise", "ES2019.Object"]
```

Comment:
Do we still need to target ES5? Only Opera Mini still doesn't support ES6.
In fact, the coverage of [`es2018.promise` support is over 96%](https://caniuse.com/promise-finally)

These polyfills are a huge portion of our bundle size. Should we keep them?

Reply:
not at all